docs/usage/config-overview.md

```diff
@@ -155,6 +155,17 @@ Inherited config may use all Repository config settings, and any Global config o
 
 For information on how the Mend Renovate App supports Inherited config, see the dedicated "Mend Renovate App Config" section toward the end of this page.
 
+#### Presets handling
+
+If the inherited config has presets, then Renovate:
+
+1. Resolves the presets
+1. Adds the resolved presets to the inherited config
+1. Merges the presets on top of the global config
+
```

Comment:
```suggestion
##### You can not ignore presets from inherited config
```

I think a sub-heading helps separate the "steps that Renovate does" from the explanation why you can not ignore inherited presets.